store/cockroach/cockroach.go

```diff
@@ -0,0 +1,113 @@
+package cockroach
+
+import (
+	"context"
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/cortezaproject/corteza-server/pkg/ql"
+	"github.com/cortezaproject/corteza-server/store"
+	"github.com/cortezaproject/corteza-server/store/rdbms"
+	"github.com/lib/pq"
+	"go.uber.org/zap"
+	"net/url"
+	"strings"
+)
+
+type (
+	Store struct {
+		*rdbms.Store
+	}
+)
+
+const (
+	storeTypeName = "cockroachdb"
+	// storeTypeWithDebug = "cockroachdb+debug"
+)
+
+// For now, CockroachDB implementation is disabled due to some limitation from cockroachDB!
+//
+//	Here is why could not work on this further:
+//	* Prefix/Expression Indexes.
+//		I.e. we are not able to use "lower" expression while creating/updating indexes
+//	* Not able to use Prefix/Expression into upsert.
+//		I.e. also not able to use "lower"(for most case) expression in upsert
+//
+// @note we could enable this once cockroachDB overcome above limitations :)
+func init() {
+	//store.Register(Connect, storeTypeName, storeTypeWithDebug)
```

Comment:
How come this is commented out?
Can we leave it enabled? Since the cockroachdb package (this) itself is not imported it does not matter what we have in the init fn()...

Reply:
Yes, I Will add it back!